torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -2877,6 +2878,79 @@ std::tuple<Tensor, Tensor> triangular_solve_backward(
   return std::tuple<Tensor, Tensor>{grad_b, grad_a};
 }
 
+Tensor linalg_solve_triangular_forward(
+    const Tensor& A_t,
```

Comment:
nit: let's add `.._forward_AD` to eliminate the confusion? Looks like this function is only used for computing the forward AD, right?